docs/_docs/windows.md

```diff
@@ -45,6 +45,7 @@ sudo gem install jekyll bundler
 You can test by running:
 
 ```
+cd ~
```

Comment:
Would it be easier to simply add a comment here saying that you should choose which directory you run the command in?